source/common/formatter/stream_info_formatter.cc

```diff
@@ -889,7 +889,11 @@ const StreamInfoFormatterProviderLookupTable& getKnownStreamInfoFormatterProvide
             [](const std::string&, absl::optional<size_t>) {
               return std::make_unique<StreamInfoUInt64FormatterProvider>(
                   [](const StreamInfo::StreamInfo& stream_info) {
-                    return stream_info.upstreamInfo()->upstreamConnectionId().value_or(0);
+                    unsigned long result = 0;
```

Comment:
We rarely use the type name `unsigned long`, may be `uint64_t` would be better here.

Reply:
@wbpcode - fixed. CI seems to be failing for an unrelated test and I can't seem to restart the test using `/retest`. Ran the same failing test locally and its succeeding, so i'm guessing its flaky ? 


```

bazel test //test/extensions/filters/http/alternate_protocols_cache:filter_integration_test

```